### PR TITLE
Add parameters on the chart's readme

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>

**Description**

Add the parameters to the charts and the sub-charts on the README.md
Bump the version of the chart.

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: install/kubernetes/helm/meshery/README.md [hunk 1]
```diff
@@ -1,98 +1,74 @@
...
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+|  | meshery-consul | 2.0.0 |
+|  | meshery-istio | 2.0.0 |
+|  | meshery-kuma | 1.0.0 |
+|  | meshery-linkerd | 2.0.0 |
+|  | meshery-nsm | 1.0.0 |
+|  | meshery-osm | 1.0.0 |
```
Comment (leecalcote) [inline install/kubernetes/helm/meshery/README.md:24]: Thank you for this, @Aisuko. This looks nice. 

The Traefik Mesh and NGINX SM adapters are ready for incorporation.

**Review by leecalcote — APPROVED** [review]
Excellent. :)

File: install/kubernetes/helm/meshery/README.md (hunk 1 above)
Comment (Aisuko) [reply install/kubernetes/helm/meshery/README.md:24]: No problems. And `meshSync`

Comment (leecalcote) [reply install/kubernetes/helm/meshery/README.md:24]: Oh, right! // @dhruv0000

**Pull request merged**